src/rmath.jl

```diff
@@ -53,10 +53,10 @@ function _import_rmath(rname::Symbol, jname::Symbol, pargs)
 
     # function implementation
     quote
-        $pdf($(pdecls...), x::Real) =
+        $pdf($(pdecls...), x::Float64) =
```

Comment:
I think it is the right thing to do. Conversion shouldn't happen in the function that `ccall`s. Also, I don't think `StatsFuns` should do much conversion at all so you can just go ahead and change the tests in https://github.com/JuliaStats/StatsFuns.jl/blob/master/test/rmath.jl#L123-L129 to use floats instead of integers.